Makefile

```diff
@@ -789,6 +796,17 @@ keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_de
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: cinder_kuttl_run
+cinder_kuttl_run: ## runs kuttl tests for the keystone operator, assumes that everything needed for running the test was deployed beforehand.
+	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${CINDER_KUTTL_CONF} ${CINDER_KUTTL_DIR}
+
+.PHONY: cinder_kuttl
+cinder_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy mariadb_deploy_validate cinder_deploy_prep keystone ## runs kuttl tests for the cinder operator. Installs openstack crds and cinder operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+	make cinder_kuttl_run
+	make deploy_cleanup
+	make cinder_cleanup
+	make mariadb_cleanup
```

Comment:
It's missing `make keystone_cleanup` between `cinder_clenaup` and `mariadb_cleanup`